.github/workflows/release.yml

```diff
@@ -4,18 +4,230 @@ on:
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload release assets
+        run: |
+          hub release edit \
+            -a ./com.x8bit.bitwarden.aab \
+            -a ./com.x8bit.bitwarden.apk \
+            -a ./com.x8bit.bitwarden-fdroid.apk \
+            -m "Version $RELEASE_NAME" \
+            $RELEASE_TAG_NAME
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RELEASE_NAME: ${{ needs.create-release.outputs.mobile-version }}
+          RELEASE_TAG_NAME: ${{ steps.latest-release-tag.outputs.release_tag_name }}
+
+      - name: Deploy to Play Store
+        run: |
+          PUBLISHER_PATH="./store/google/Publisher/bin/Release/netcoreapp2.0/Publisher.dll"
+          CREDS_PATH="$HOME/secrets/play_creds.json"
+          AAB_PATH="./com.x8bit.bitwarden.aab"
+          TRACK="internal"
+
+          dotnet $PUBLISHER_PATH $CREDS_PATH $AAB_PATH $TRACK
+        shell: bash
+
 
-  cloc:
+  android-ubuntu:
+    name: Android Ubuntu Build
     runs-on: ubuntu-latest
+    needs: android
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # v2.3.4
+        with:
+          ref: rc
+
+      - name: Download F-Droid .apk artifact
+        uses: dawidd6/action-download-artifact@b9571484721e8187f1fd08147b497129f8972c74 # v2.14.0
+        with:
+          workflow: build.yml
+          workflow_conclusion: success
+          branch: rc
+          name: com.x8bit.bitwarden-fdroid.apk
+
+      - name: Set up Node
+        uses: actions/setup-node@46071b5c7a2e0c34e49c3cb8a0e792e86e18d5ea # v2.3.0
+        with:
+          node-version: '10.x'
+
+      - name: Set up F-Droid server
+        run: |
+          sudo apt-get -qq update
+          sudo apt-get -qqy install --no-install-recommends fdroidserver wget
+
+      - name: Set up Git credentials
+        env:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        run: |
+          git config --global credential.helper store
+          echo "https://${ACCESS_TOKEN}:x-oauth-basic@github.com" >> ~/.git-credentials
+          git config --global user.email "ci@bitwarden.com"
+          git config --global user.name "Bitwarden CI"
+
+      - name: Print environment
+        run: |
+          node --version
+          npm --version
+          git --version
+          echo "GitHub ref: $GITHUB_REF"
+          echo "GitHub event: $GITHUB_EVENT"
+
+      - name: Install Node dependencies
+        run: npm install
 
+      - name: Decrypt secrets
+        run: |
+          mkdir -p ~/secrets
+          gpg --quiet --batch --yes --decrypt --passphrase="$DECRYPT_FILE_PASSWORD" --output ./store/fdroid/keystore.jks ./.github/secrets/store_fdroid-keystore.jks.gpg
+        env:
+          DECRYPT_FILE_PASSWORD: ${{ secrets.DECRYPT_FILE_PASSWORD }}
+
+      - name: Compile for F-Droid Store
+        run: |
```

Comment:
Is there an easy way to move this compilation to the build portion of the pipeline?